EDDiscovery/UserControls/UserControlScanGrid.Designer.cs

```diff
@@ -0,0 +1,178 @@
+            this.dataViewScrollerPanel2 = new ExtendedControls.DataViewScrollerPanel();
+            this.dataGridViewScangrid = new System.Windows.Forms.DataGridView();
+            this.vScrollBarCustom2 = new ExtendedControls.VScrollBarCustom();
+            this.img = new System.Windows.Forms.DataGridViewImageColumn();
+            this.BodyName = new System.Windows.Forms.DataGridViewTextBoxColumn();
+            this.BodyClass = new System.Windows.Forms.DataGridViewTextBoxColumn();
+            this.BodyDetails = new System.Windows.Forms.DataGridViewTextBoxColumn();
+            this.dataViewScrollerPanel2.SuspendLayout();
+            ((System.ComponentModel.ISupportInitialize)(this.dataGridViewScangrid)).BeginInit();
+            this.SuspendLayout();
+            // 
+            // dataViewScrollerPanel2
+            // 
+            this.dataViewScrollerPanel2.Controls.Add(this.dataGridViewScangrid);
+            this.dataViewScrollerPanel2.Controls.Add(this.vScrollBarCustom2);
+            this.dataViewScrollerPanel2.Dock = System.Windows.Forms.DockStyle.Fill;
+            this.dataViewScrollerPanel2.InternalMargin = new System.Windows.Forms.Padding(0);
+            this.dataViewScrollerPanel2.Location = new System.Drawing.Point(0, 0);
+            this.dataViewScrollerPanel2.Name = "dataViewScrollerPanel2";
+            this.dataViewScrollerPanel2.ScrollBarWidth = 20;
+            this.dataViewScrollerPanel2.Size = new System.Drawing.Size(572, 572);
+            this.dataViewScrollerPanel2.TabIndex = 25;
+            this.dataViewScrollerPanel2.VerticalScrollBarDockRight = true;
+            // 
+            // dataGridViewScangrid
+            // 
+            this.dataGridViewScangrid.AllowUserToAddRows = false;
+            this.dataGridViewScangrid.AllowUserToDeleteRows = false;
+            this.dataGridViewScangrid.AllowUserToResizeRows = false;
+            this.dataGridViewScangrid.AutoSizeColumnsMode = System.Windows.Forms.DataGridViewAutoSizeColumnsMode.Fill;
+            this.dataGridViewScangrid.ColumnHeadersHeightSizeMode = System.Windows.Forms.DataGridViewColumnHeadersHeightSizeMode.AutoSize;
+            this.dataGridViewScangrid.Columns.AddRange(new System.Windows.Forms.DataGridViewColumn[] {
+            this.img,
+            this.BodyName,
+            this.BodyClass,
+            this.BodyDetails});
+            this.dataGridViewScangrid.Dock = System.Windows.Forms.DockStyle.Bottom;
+            this.dataGridViewScangrid.Location = new System.Drawing.Point(0, 0);
+            this.dataGridViewScangrid.Name = "dataGridViewScangrid";
+            this.dataGridViewScangrid.RowHeadersVisible = false;
+            this.dataGridViewScangrid.RowTemplate.Height = 35;
+            this.dataGridViewScangrid.ScrollBars = System.Windows.Forms.ScrollBars.None;
+            this.dataGridViewScangrid.Size = new System.Drawing.Size(552, 572);
+            this.dataGridViewScangrid.TabIndex = 23;
```

Comment:
I don't see any `ReadOnly` flags set. Is it possible for users to edit the scan data through this Control in any way? If not, is it still possible for them to access the cell edit controls, even if the underlying data type is read-only?

Reply:
Sorry, my fault!

Reply:
Done.